src/Microsoft.DotNet.Darc/src/DarcLib/GitHubClient.cs

```diff
@@ -174,6 +174,13 @@ public async Task CreateBranchAsync(string repoUri, string newBranch, string bas
             }
         }
 
+        public async Task DeleteBranchAsync(string repoUri, string branch)
+        {
+            (string owner, string repo) = ParseRepoUri(repoUri);
+
+            await Client.Git.Reference.Delete(owner, repo, $"heads/{branch}");
```

Comment:
Should this be refs/heads?

Reply:
I had refs/heads at first but then I looked at how we are currently deleting the branch after merging and we do it like that. https://github.com/dotnet/arcade-services/blob/master/src/Microsoft.DotNet.Darc/src/DarcLib/GitHubClient.cs#L344